src/open_inwoner/openzaak/models.py

```diff
@@ -300,6 +302,63 @@ def __str__(self):
         return self.omschrijving
 
 
+class ZaakTypeStatusTypeConfig(models.Model):
+    zaaktype_config = models.ForeignKey(
+        "openzaak.ZaakTypeConfig",
+        on_delete=models.CASCADE,
+    )
+    statustype_url = models.URLField(
+        verbose_name=_("Statustype URL"),
+        max_length=1000,
+    )
+    omschrijving = models.CharField(
+        verbose_name=_("Omschrijving"),
+        max_length=80,
+    )
+    statustekst = models.CharField(
+        verbose_name=_("Statustekst"),
+        max_length=1000,
+    )
+    zaaktype_uuids = ArrayField(
```

Comment:
I'm not sure why this is in there as we already have a relation with a zaaktype_config. But as this is done this way for the ZTIOT-Config I guess there is a purpose...